packages/@aws-cdk/aws-ecs/lib/firelens-log-router.ts

```diff
@@ -116,7 +117,7 @@ function renderFirelensConfig(firelensConfig: FirelensConfig): CfnTaskDefinition
       options: {
```

Comment:
I think we can also remove the `if-else` condition here to always return the firelens config in the `else` part (as the `enable-ecs-log-metadata` in the `options` will always be present now).